cmd/client.go

```diff
@@ -33,3 +40,23 @@ func createClient(ctx context.Context, host string) (shieldv1beta1.ShieldService
 	client := shieldv1beta1.NewShieldServiceClient(conn)
 	return client, cancel, nil
 }
+
+func isClientCLI(cmd *cobra.Command) bool {
```

Comment:
@mabdh good for now, in future this method check of annotations can be moved to salt.